lib/libvarnish/vsub.c

```diff
@@ -67,13 +65,14 @@ VSUB_closefrom(int fd)
 
 	assert(fd >= 0);
 
-#ifdef HAVE_WORKING_CLOSE_RANGE
-	AZ(close_range(fd, ~0U, 0));
-	return;
-#elif HAVE_CLOSEFROM
+#ifdef HAVE_CLOSEFROM
 	closefrom(fd);
 	return;
 #else
+#ifdef HAVE_WORKING_CLOSE_RANGE
+	if (close_range(fd, ~0U, 0) == 0)
+		return;
+#endif
```

Comment:
Aren't you leaving a stray `#endif` below?

Reply:
No, its nested ifdef to avoid the include of `<dirent.h>` . Readability isn't nice, it would be more readable to always include dirent and just ifdef the `HAVE_CLOSEFROM`

Reply:
Indented to make it more readable